integration/side-effects/snapshots/esm2015/testing.js

```diff
@@ -0,0 +1,7 @@
+var NotificationKind;
+
+(function(NotificationKind) {
+    NotificationKind["NEXT"] = "N";
+    NotificationKind["ERROR"] = "E";
+    NotificationKind["COMPLETE"] = "C";
+})(NotificationKind || (NotificationKind = {}));
```

Comment:
This code is retained because of how TS transpiles enums. Newer versions of TS don't suffer from this.